app/jobs/update_dtc.rb

```diff
@@ -0,0 +1,34 @@
+require 'open-uri'
+
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
+
+class UpdateDtc < TsvUpdater
+  def tempfile_name
+    ['dtc_bioactivity', '.csv']
+  end
+
+  def importer
+    Genome::Importers::Dtc::DtcImporter.new(tempfile)
+  end
+
+  def download_file
+    uri = URI('https://drugtargetcommons.fimm.fi/static/Excell_files/DTC_data.csv')
+    download = open(uri)
+    IO.copy(download, tempfile_name)
+  end
+end
```

Comment:
These `end`s here (lines 19-21 and line 34) are extra. You will also want to indent all of the `should` methods to be on the same level as the rest of the methods.